src/app/services/communibee-backend/category/category.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { BackendModelService } from '../basic-model.service';
+import { Category, CategoryModel } from './category';
+import { BeckendHttpService } from '../beckend-http.service';
+
+const path = 'categories';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CategoryService extends BackendModelService<Category, CategoryModel> {
+
+  constructor(private communiHttp: BeckendHttpService) {
```

Comment:
Rename `communiHttp` to a more proper name (for example, `backendHttpService` is fine).
(This applies for other files, too)